db/schema.rb

```diff
@@ -94,6 +94,27 @@
     t.datetime "updated_at",  null: false
   end
 
+  create_table "contributions", force: :cascade do |t|
+    t.integer  "article_id"
```

Comment:
I looks like maybe this came in from your other branch with roles/contributors.

I think you can `rake db:rollback` once or twice to get it back into a good state.
Or there's almost certainly a `git` way to roll back just one file, but it's alluding me right now.